src/main/java/org/zalando/jackson/datatype/money/MonetaryAmountSerializer.java

```diff
@@ -16,6 +16,7 @@
 
     private final MonetaryAmountFormatFactory factory;
     private final FieldNames names;
+    private final boolean numberAsString;
```

Comment:
Do you mean own or Jackson's? With Jackson's we have to fail if someone requests other feature beside `WRITE_NUMBERS_AS_STRINGS`